sdks/python/examples/bulk_operations/test_bulk_replay.py

```diff
@@ -90,8 +90,8 @@ async def test_bulk_replay(hatchet: Hatchet) -> None:
 
     for run in runs.rows:
         assert run.status == V1TaskStatus.COMPLETED
-        assert run.retry_count == 1
-        assert run.attempt == 2
+        assert run.retry_count >= 1
+        assert run.attempt >= 2
```

Comment:
this was flaky - it was getting broken because of the worker backlog fix from a few weeks ago, because there would be a failure to assign which would increment the retry count, so we'd end up with a retry count of e.g. 2 instead of 1. I think it's fine to make this less specific